WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -139,111 +140,120 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager openSansBoldFontOfSize:12.0];
 }
 
+
 #pragma mark - Colors
+// https://wordpress.com/design-handbook/colors/
 
-+ (UIColor *)wordPressBlue
++ (UIColor *)colorWithR:(NSInteger)red G:(NSInteger)green B:(NSInteger)blue alpha:(CGFloat)alpha
```

Comment:
> but I think it makes readability worse in the file.
Really? For me its so much easier to understand RGB values when I can just focus on the number, and not have to filter out all the repetitious division.